app/src/components/TimeLine.js

```diff
@@ -1,18 +1,29 @@
 import React, { useState, useEffect } from "react";
 
+import dayjs from "dayjs";
+
 function TimeLine({ timeRows, isAuthenticated, dayStart, totalHours }) {
   //Returns an array of time objects at half-hour increments with row location
   //based on dayStart starting hour and totalHours
   function getTimes() {
     let timesToRender = [];
-    for (let i = 0; i <= totalHours; i += 0.5) {
-      let timeString = `${new Date(dayStart).getHours() + Math.floor(i)}`;
-      if (i % 1 !== 0) {
-        timeString += ":30";
+    for (let i = 0; i <= totalHours; i += 0.25) {
+      let value = dayjs(dayStart).add(Math.floor(i), "hour"); //creates dayjs object, determines hour
+      let minute;
+      if (i % 1 === 0.25) {
+        minute = 15;
+      } else if (i % 1 === 0.5) {
+        minute = 30;
+      } else if (i % 1 === 0.75) {
+        minute = 45;
+      } else {
+        minute = 0;
       }
-      let timeLineLocation = `${i * 2 + 1}`;
+      value = value.set("minute", minute); //sets minute
+
+      let timeLineLocation = `${i * 4 + 1}`; //determines row to render on
 
-      timesToRender.push({ timeString, timeLineLocation });
+      timesToRender.push({ value, timeLineLocation }); //pushes as object
```

Comment:
The comment feels a lil extra/unneeded since the `{...}` already indicates a JS object.